ZenPacks/zenoss/ZenPackLib/lib/zuul.py

```diff
@@ -59,12 +59,13 @@
 from Products.Zuul.facades.processfacade import ProcessFacade
 from Products.Zuul.facades.servicefacade import ServiceFacade
 
-from Products.Zuul.catalog.paths import (
-     DevicePathReporter,
+from Products.Zuul.catalog.paths import DevicePathReporter
```

Comment:
This and the following change make no functional difference, and they're the only changes to this file. It would be better to leave them out of this commit. They only distract from the real changes.